emsdk.py

```diff
@@ -116,6 +116,9 @@ def errlog(msg):
   ARCH = 'x86'
 elif machine.startswith('aarch64') or machine.lower().startswith('arm64'):
   ARCH = 'aarch64'
+  if MACOS:
+    # arm64 macOS runs x86_64 binaries and we don't have build bots yet for native
+    ARCH = 'x86_64'
```

Comment:
Can we instead have a command line option to opt into this?  `--arch=x86_64`

Or an environment variable `EMSDK_ARCH=x86_64`?

With #753 we not support native python and node, although it only works in "build from source" mode.   This change as it stand would render that path unavailable, no?

Reply:
I don't think we should use a command-line option; rather it should "just work". However using the Aarch64 node and python would be much preferable. What would you recommend for fetching the llvm & binaryen binaries for "x64" while fetching node and python from Aarch64?

Reply:
(You *are* using binaries of python and node right? Or do I misunderstand and those are from source too?)

Reply:
If it's possible to build the macOS binaries as universal binaries with the current SDK, that would be nice too -- then we wouldn't have to do *any* of this except fetching Aarch64-capable node and python.

Reply:
Does anyone know the *exact* build scripts used to produce the llvm/clang and binaryen binary artifacts? I would like to modify them to resolve this once and for all. ;)

Reply:
(Note that Node builds as aarch64 on mac, but they're still working on official universal binaries...)

Reply:
Universal binaries also double the download side right... I'm not sure we need/want that.  Why not just build twice and upload two different archives?